cfgov/jinja2/v1/_includes/sidenav.html

```diff
@@ -22,25 +22,25 @@
 
 {% macro render(items, active_item_id, additional_classes) %}
 
-<nav class="nav-secondary expandable {{ additional_classes }}"
+<nav class="nav-secondary expandable {{ additional_classes if additional_classes else '' }}"
      aria-label="Section navigation">
     <div class="expandable_header nav-secondary_header">
-        <button class="expandable_target nav-secondary_link nav-secondary_link__button">
+        <button class="expandable_target nav-secondary_button">
             <span class="expandable_header-left">
                 In this section
             </span>
             <span class="expandable_header-right">
                 <span class="expandable_cue-open">
-                    <span class="cf-icon cf-icon-down cf-icon__green"></span>
+                    <span class="cf-icon cf-icon-down"></span>
                 </span>
                 <span class="expandable_cue-close">
-                    <span class="cf-icon cf-icon-up cf-icon__green"></span>
+                    <span class="cf-icon cf-icon-up"></span>
                 </span>
             </span>
         </button>
     </div>
     <div class="expandable_content nav-secondary_content">
-        <ul class="nav-secondary_list">
+        <ul class="nav-secondary_list nav-secondary_list__parents">
```

Comment:
Should this be `parents` or `parent`? I think the modifier is `__parents` in the css but singular [here](https://github.com/cfpb/cfgov-refresh/pull/1144/files#diff-b745345cfdbaf1327c1cb3a2a77a4d1eR34).